crawlers/related_news_crawler.py

```
@@ -0,0 +1,79 @@
+    feed = feedparser.parse(rss_feed[0])
+    print("RSS URL:", rss_feed[0])
+
+    print("Feed keys:", feed.keys())  # feed.entries 외에 어떤 키가 있는지 보기
+    print("Entries 개수:", len(feed.entries))
+
+    print(f"RSS에서 {len(feed.entries)}개의 기사 발견!")
+
+    # 2️⃣ 각 기사 크롤링
+    sqlList = []
+    paramList = []
+    crawl_count = 10
```

Pylint (3.3.7)

[convention] Constant name "crawl_count" doesn't conform to UPPER_CASE naming style

(C0103)